### ISSUE TITLE
ESP32-S3-BOX

### ISSUE THREAD

**Issue by djschwab** [pschatzmann/arduino-audiokit#79]
Will this library work with board type ESP32-S3-BOX? I set 
"#  define AUDIOKIT_BOARD 11"
and get the following compiler errors:

In file included from /tmp/.arduinoIDE-unsaved202344-707384-8d5iyl.1o9ow/output/output.ino:10:
/home/dave/Arduino/libraries/audiokit/src/AudioKitHAL.h:34:19: error: 'VSPI' was not declared in this scope
 SPIClass SPI_VSPI(VSPI);
                   ^~~~
/home/dave/Arduino/libraries/audiokit/src/AudioKitHAL.h:34:19: note: suggested alternative: 'SPI'
 SPIClass SPI_VSPI(VSPI);
                   ^~~~
                   SPI
/home/dave/Arduino/libraries/audiokit/src/AudioKitHAL.h: In member function 'i2s_mode_t audiokit::AudioKitConfig::i2sMode()':
/home/dave/Arduino/libraries/audiokit/src/AudioKitHAL.h:199:39: error: 'I2S_MODE_DAC_BUILT_IN' was not declared in this scope
           mode = mode | I2S_MODE_TX | I2S_MODE_DAC_BUILT_IN;
                                       ^~~~~~~~~~~~~~~~~~~~~
/home/dave/Arduino/libraries/audiokit/src/AudioKitHAL.h:199:39: note: suggested alternative: 'I2S_MODE_MASTER'
           mode = mode | I2S_MODE_TX | I2S_MODE_DAC_BUILT_IN;
                                       ^~~~~~~~~~~~~~~~~~~~~
                                       I2S_MODE_MASTER
/home/dave/Arduino/libraries/audiokit/src/AudioKitHAL.h:201:39: error: 'I2S_MODE_ADC_BUILT_IN' was not declared in this scope
           mode = mode | I2S_MODE_RX | I2S_MODE_ADC_BUILT_IN;
                                       ^~~~~~~~~~~~~~~~~~~~~
/home/dave/Arduino/libraries/audiokit/src/AudioKitHAL.h:203:53: error: 'I2S_MODE_ADC_BUILT_IN' was not declared in this scope
           mode = mode | I2S_MODE_RX | I2S_MODE_TX | I2S_MODE_ADC_BUILT_IN | I2S_MODE_DAC_BUILT_IN;
                                                     ^~~~~~~~~~~~~~~~~~~~~
/home/dave/Arduino/libraries/audiokit/src/AudioKitHAL.h:203:77: error: 'I2S_MODE_DAC_BUILT_IN' was not declared in this scope
           mode = mode | I2S_MODE_RX | I2S_MODE_TX | I2S_MODE_ADC_BUILT_IN | I2S_MODE_DAC_BUILT_IN;
                                                                             ^~~~~~~~~~~~~~~~~~~~~
/home/dave/Arduino/libraries/audiokit/src/AudioKitHAL.h:203:77: note: suggested alternative: 'I2S_MODE_MASTER'
           mode = mode | I2S_MODE_RX | I2S_MODE_TX | I2S_MODE_ADC_BUILT_IN | I2S_MODE_DAC_BUILT_IN;
                                                                             ^~~~~~~~~~~~~~~~~~~~~
                                                                             I2S_MODE_MASTER

exit status 1

Compilation error: exit status 1

**Comment by djschwab** [comment]
Yes it is. v2.1.0. I was trying to compile the example 'output.ino'.

**Comment by pschatzmann** [comment]
You have the following options to resolve this:
- Deactivate the SD 
- select the generic module from the Arduino menu 
- upgrade the library and test again...

**Comment by pschatzmann** [comment]
Unfortunatly I do not have an info about this board, so to make this work you will need to double check the hardware spec and define the pins!

**Comment by pschatzmann** [comment]
I was hoping you pick this up, since I can't test this myself w/o the corresponding board.
I committed a DRAFT implementation.

**Comment by djschwab** [comment]
Thanks Phil!
Output works OK. Input only gives zeroes.
My issue is solved. I was only concerned about output at this stage.
Thanks again.
